tests/unit/s2n_cttime_functions_test.c

```diff
@@ -0,0 +1,40 @@
+/*
```

Comment:
No, I hadn't seen that test, it does indeed cover everything tested by this one.
Sorry, I was over-enthusiastic when I saw that work was being done on constant-time operations in s2n, and I submitted a terrible branch. Perhaps keep the first commit only?